blueprints/app/files/.eslintignore

```diff
@@ -8,6 +8,7 @@
 
 # dependencies
 /bower_components/
+/node_modules/
```

Comment:
or should we do `/!.*` below instead?

Reply:
I'm not sure if that maybe breaks some things which were fixed in #8117. E.g if you had a `/tests/.foo.js` file..

Reply:
I think we need to add !/node_modules ?

Reply:
@Gaurav0 Why would we want to **not** ignore `node_modules`?

Reply:
Sorry meant specifically add node_modules.

Reply:
I've included this in my PR and working on fixing all tests.